### PR TITLE
docs: add oracle overview and modify oracle part

### PR DESCRIPTION
**Pull request by audtlr24**
Add `about-dep-oracle` and modify `operate-oracle-nodes` part to match the current spec.

### CODE REVIEW

File: .gitbook/5-oracles/0-about-dep-oracle.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+Oracle is a validator that validates data as it transmits from provider to consumer.
+
+Since data encryption is required for data privacy, `Panacea`, a public blockchain, cannot act as a validator.
+Instead, the oracle decrypts and validates the provider's data and provides the result in the form of a `certificate`.
+Then, data consumer and `Panacea blockchain` can check the `certificate` to determine the validity of the data.
+
+For a detailed process, see [Data Validation](../3-protocol-devs/1-dep-specs/4-data-validation.md#data-validation).
+
+
+### Mediator of Data Delivery
```
Comment (youngjoon-lee) [inline .gitbook/5-oracles/0-about-dep-oracle.md:21]:
```suggestion
### Mediator for Data Delivery
```

File: .gitbook/5-oracles/0-about-dep-oracle.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+Oracle also works as a data deliverer.
+The oracle is responsible for delivering the verified data to the appropriate consumer.
+And, to prevent malicious consumers who just want to get the data without paying the incentive, the oracle is responsible for managing the data decryption key (`secret key`).
+The consumer can get `secret key` through the oracle after the provider get incentive.
+
+For a detailed process, see [Data Validation](../3-protocol-devs/1-dep-specs/4-data-validation.md#data-re-encryption-and-delivery-via-consumer-service).
+
+## Oracle using SGX
```
Comment (youngjoon-lee) [inline .gitbook/5-oracles/0-about-dep-oracle.md:30]:
```suggestion
## Oracle based on SGX
```

**Review by youngjoon-lee — APPROVED** [review]
lgtm!

File: .gitbook/5-oracles/0-about-dep-oracle.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+Oracle is a key component of `data validation` in the `DEP` ecosystem.
+
+### Data Validator
+
+Oracle is a validator that validates data as it transmits from provider to consumer.
+
+Since data encryption is required for data privacy, `Panacea`, a public blockchain, cannot act as a validator.
+Instead, the oracle decrypts and validates the provider's data and provides the result in the form of a `certificate`.
+Then, data consumer and `Panacea blockchain` can check the `certificate` to determine the validity of the data.
```
Comment (gyuguen) [inline .gitbook/5-oracles/0-about-dep-oracle.md:16]: We called blockchain a `Panacea` above, so let's just call it `Panacea` here too
```suggestion
Then, data consumer and `Panacea` can check the `certificate` to determine the validity of the data.
```

**Review by gyuguen — APPROVED** [review]
LGTM

**Pull request merged**